Docs: V7 update guide needs to mention minimum dependencies

<!-- ❤️ ngrx? Please consider supporting our collective: 👉  [donate](https://opencollective.com/ngrx/donate) -->

## Support Requests should not be opened as issues and should be handled in the following ways:

### On [StackOverflow](https://stackoverflow.com/questions/tagged/ngrx) using the `ngrx` tag

### In our gitter support channel at https://gitter.im/ngrx/platform

ERROR in node_modules/@ngrx/store/src/models.d.ts(31,43): error TS1005: ';' expected. node_modules/@ngrx/store/src/models.d.ts(31,67): error TS1005: ',' expected.

--- a/projects/ngrx.io/content/guide/migration/v7.md
+++ b/projects/ngrx.io/content/guide/migration/v7.md
@@ -12,7 +12,12 @@ ng update @ngrx/store
 
 ## Dependencies
 
-NgRx now has a minimum version requirement on Angular 7.
+V7 has the minimum version requirements:
+
+- Angular version 7
+- TypeScript 3.1.x
+- RxJS 6.x
+
 
 ## @ngrx/store
 

--- a/projects/ngrx.io/content/guide/migration/v7.md
+++ b/projects/ngrx.io/content/guide/migration/v7.md
@@ -16,7 +16,7 @@ V7 has the minimum version requirements:
 
 - Angular version 7
 - TypeScript version 3.1.x
-- RxJS 6.x
+- RxJS version 6.x
 
 
 ## @ngrx/store

--- a/projects/ngrx.io/content/guide/migration/v7.md
+++ b/projects/ngrx.io/content/guide/migration/v7.md
@@ -10,6 +10,10 @@ To update your packages to the latest released version, run the command below.
 ng update @ngrx/store
 ```
 
+## Dependencies
+
+NgRx now has a minimum version requirement on Angular 7.
+
 ## @ngrx/store
 
 ### Feature loaded action

--- a/projects/ngrx.io/content/guide/migration/v7.md
+++ b/projects/ngrx.io/content/guide/migration/v7.md
@@ -15,7 +15,7 @@ ng update @ngrx/store
 V7 has the minimum version requirements:
 
 - Angular version 7
-- TypeScript 3.1.x
+- TypeScript version 3.1.x
 - RxJS 6.x
 
 

